docs/application/native/guides/device/system.md

```diff
@@ -730,6 +730,15 @@ The following table lists the build system keys:
 | `http://tizen.org/system/build.variant`  | `String` | The platform returns the variant release information. The variant release information is made when the platform image is created. |
 | `http://tizen.org/system/build.release` | `String` | The platform returns the build version information. The build version information is made when the platform image is created. |
 
+The following table lists the device keys:
```

Comment:
Question. Don't we need to add a link to this table on line 73?

Reply:
I just add a link like other keys.
If you have a better idea (where this key include), please tell me.

Meanwhile, table name is modified (device -> device type) because it is not device.type but device_type.

Reply:
I can see `<a name="devicetype"></a>` is applied well, but for now "#devicetype" is called from nowhere.
So I suggested to add a link to the list in the above of this page like other tables.
![image](https://user-images.githubusercontent.com/9958374/57742942-0047ff80-76fe-11e9-987f-c0b1da64e658.png) 

The md file, line 73.
![image](https://user-images.githubusercontent.com/9958374/57742976-31c0cb00-76fe-11e9-9c7f-6ead8856dab0.png)

I think the the location of this table is OK.

Reply:
Modified.